source/parser.hera

```diff
@@ -4214,62 +4214,57 @@ Reset
       verbose: false,
     }
 
-    let indexOfRef, hasPropRef, spliceRef
-    const asAny = {
+    module.asAny = {
       ts: true,
       children: [" as any"]
     }
     module.prelude = []
 
-    module.getSpliceRef = function () {
-      if (spliceRef) return spliceRef
-
-      spliceRef = {
-        type: "Ref",
-        base: "splice",
-        id: "splice",
-      }
-
-      return spliceRef
+    const refMakers = {
```

Comment:
Maybe name this `addRefDeclaration` instead of `refMakers`?

Reply:
Good idea. How about `declareRef`?